DataManagementSystem/scripts/dirac-dms-directory-sync.py

```diff
@@ -111,7 +112,7 @@ def getSetOfRemoteSubDirectoriesAndFiles(path,fc,directories,files):
           return S_ERROR('Error: ' + res['Message'])
       return S_OK()
     else:
-      return S_ERROR("Error:" + result['Message'])
+      return S_ERROR("Error: %s" % result['Value'])
```

Comment:
Well, the bug here was that the result is not S_ERROR at this point and therefore there is no `Message`. Doing `return result` would not be good in this case.